06_puppy/kasunfdo/types.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"sync"
+)
+
+// Data structure for puppy
+type Puppy struct {
+	ID     uint64
+	Breed  string
+	Colour string
+	Value  float64
+}
+
+// Storer defines standard CRUD operations for Puppy
+type Storer interface {
+	CreatePuppy(puppy *Puppy) bool
+	ReadPuppy(id uint64) (*Puppy, bool)
+	UpdatePuppy(id uint64, puppy *Puppy) bool
+	DeletePuppy(id uint64) bool
+}
+
+// MapStore type and implementation
```

Comment:
This is also not the correct format for a doc string. Furthermore it is wrong because there is no implementation here.